### PR TITLE
Feature/115 mapping pcr

### PR DESCRIPTION
**Pull request by ErikTute**
Tests and CI both pass

### CODE REVIEW

**Comment by SevKohler** [comment]
Please update to newest develop

**Comment by ErikTute** [comment]
@SevKohler updated as requested

File: src/main/java/org/ehrbase/fhirbridge/ehr/converter/specific/virologischerBefund/PCRObservationConverter.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    protected BefundObservation convertInternal(Observation resource) {
+        BefundObservation befundObservation = new BefundObservation();
+        befundObservation.setFeederAudit(createIdentifierSection(resource));
+        befundObservation.setLabortestBezeichnungDefiningCode(LabortestBezeichnungDefiningCode.DETECTION_OF_VIRUS_PROCEDURE);
+        befundObservation.setLabortestPanel(createLabortestPanel(resource));
+        return befundObservation;
+    }
+
+    private FeederAudit createIdentifierSection(Observation observation) {
+        FeederAudit feederAudit = new FeederAudit();
```
Comment (SevKohler) [inline src/main/java/org/ehrbase/fhirbridge/ehr/converter/specific/virologischerBefund/PCRObservationConverter.java:30]: delete feeder audit. This is automatically done by the generic classes

**Review by SevKohler — CHANGES_REQUESTED** [review]

File: src/main/java/org/ehrbase/fhirbridge/ehr/converter/specific/virologischerBefund/PCRObservationConverter.java (hunk 1 above)
Comment (ErikTute) [reply src/main/java/org/ehrbase/fhirbridge/ehr/converter/specific/virologischerBefund/PCRObservationConverter.java:30]: @SevKohler this is not the "normal" feederAudit stuff, it's part of the mapping.

Comment (SevKohler) [reply src/main/java/org/ehrbase/fhirbridge/ehr/converter/specific/virologischerBefund/PCRObservationConverter.java:30]: Please ensure that this information is contained in the mapped Composition, as i see it its overwritten by the generic classes ?

Comment (ErikTute) [reply src/main/java/org/ehrbase/fhirbridge/ehr/converter/specific/virologischerBefund/PCRObservationConverter.java:30]: Just checked the JSON used to compare in tests. It contains the ID in FeederAudit as expected and documented.  
`` 
"feeder_audit" : {
      "_type" : "FEEDER_AUDIT",
      "originating_system_item_ids" : [ {
        "_type" : "DV_IDENTIFIER",
        "assigner" : "Organization/Charité",
        "id" : "94500-6_SARS-CoV-2-RNA-Presence-in-Respiratory-specimen",
        "type" : "org.hl7.fhir.r4.model.CodeableConcept"
      } ],
``

Comment (SevKohler) [reply src/main/java/org/ehrbase/fhirbridge/ehr/converter/specific/virologischerBefund/PCRObservationConverter.java:30]: So as far as i can see it everything is contained.

**Pull request merged**